vendor/shims/-jquery.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+  var jq = self['$'].noConflict();
```

Comment:
here we heavily rely on the assumption that we included jquery right a moment ago 
https://github.com/san650/ember-cli-page-object/pull/325/files#diff-168726dbe96b3ce427e7fedce31bb0bcR35

this way `noConflict` gives us our jquery instance and disposes it from the `window`.

Reply:
I think I'll inline some comments in code a little bit later